tap_recharge/schema.py

```diff
@@ -20,6 +21,21 @@ def get_schemas():
     field_metadata = {}
 
     for stream_name, stream_object in STREAMS.items():
+
+        # The Payment Methods resource is not publicly available to every Merchant at this point.
+        # Reference: https://developer.rechargepayments.com/2021-11/payment_methods/payment_methods_list
+        # Thus checking if we have access to "payment_methods"
+        params = stream_object.params
+        params.update({'limit': 1}) # add limit 1 to only fetch 1 record
```

Comment:
If it is planned for a specific stream then can we add if condition? else we will make an API call for each stream.

Reply:
Got it. Added condition to do API call for only **payment_methods**.